dotcom-rendering/src/web/experiments/tests/recipe-structured-data.ts

```diff
@@ -0,0 +1,68 @@
+// This is for an experiment testing SEO performance on recipes before
+// and after structured data is added. Please don't delete - it will be
+// removed once the the experiment is finished, we swear!
+
+export type RecipeSchemaObject = {
+	[url: string]: string;
+};
+
+export const recipeSchema: RecipeSchemaObject = {
```

Comment:
Thinking some more, do we need this type? I'm wondering if having it has any value 🤔

Reply:
TypeScript kicks up a fuss without it I'm afraid

Reply:
Gotcha